server/container/guice/guice-common/src/main/java/org/apache/james/utils/RetryExecutorUtil.java

```diff
@@ -23,15 +23,16 @@
 
 public class RetryExecutorUtil {
 
-    public static final int INITIAL_DELAY_MILLIS = 500;
-    public static final int MULTIPLIER = 2;
-
+    private static final int INITIAL_DELAY_MILLIS = 5000;
+    private static final int MULTIPLIER = 2;
+    private static final int MAX_DELAY_MILLIS = 5000 * 1024;
```

Comment:
We used ExponentialBackoff which calculate delay time by INITIAL_DELAY_MILLIS*the power of MULTIPLIER and current retry times.
To avoid delay time even greater then MAX_DELAY_MILLIS then I set it to the max value that we can have
> 5s between each retries seems a lot, I prefer 500.

Yes, I know it to much but with cassandra it even longer. Cassandra had its retry with the power of 2 and 2000 so I would like to increase that init delay time so that elasticsearch can have more time for retrying.